main.c

```diff
@@ -544,7 +545,7 @@ void pwm_init(void)
  
 void updatePulseWidth(float percent)
 {
-    uint32_t pulseWidth_us = (uint32_t) (MOTOR_MIN_PULSE_WIDTH_US + percent * (MOTOR_MAX_PULSE_WIDTH_US - MOTOR_MIN_PULSE_WIDTH_US));
+    uint32_t pulseWidth_us = (uint32_t) (MOTOR_MIN_PULSE_WIDTH_US + ((1-percent)*0.65) * (MOTOR_MAX_PULSE_WIDTH_US - MOTOR_MIN_PULSE_WIDTH_US));
```

Comment:
Rather than scaling it here, please scale it using the min/max degrees defined at the top of the file. Doing the inversion here I think is fine